Astro CLI release notes 1.9 (#1735)
astronomer/docs: astro/cli/astro-run.md

| Option                  | Description                                                                                                                                 | Possible Values                            |
| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |

> | `-d`, `--dag-file` | The location of your DAG file. When this flag is specified, the CLI skips parsing all other DAGs in your project except for the specified DAG. | Any valid DAG file in your `dags` directory. |

Reviewer: ```suggestion
| `-d`, `--dag-file` | The location of your DAG file. When you specify this flag, only the specified DAG is parsed by the Astro CLI. All other DAGs in the project are ignored.| Any valid DAG file in your `dags` directory. |
```
